package.json

```diff
@@ -95,7 +95,7 @@
     "mocha": "^7.1.2",
     "mongodb-js-precommit": "^2.0.0",
     "mongodb-runner": "^4.7.5",
-    "node-codesign": "durran/node-codesign",
+    "node-codesign": "github:addaleax/node-codesign",
```

Comment:
This is just so that the `entitlements` option becomes available … this should probably become some official npm package at some point, I might ping Durran about it after this is merged :)

Reply:
yea i was looking at that before, and we should just publish it.

Reply:
Yeah, I’ll ping Durran and if nothing speaks against it I’ll move this into the org (i.e. move it back, but as `node-codesign`) and publish it :)